apps/roboshield/src/assets/icons/menu-icon.svg

```diff
@@ -1,5 +1,4 @@
-<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M4 24H28" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-<path d="M4 16H28" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-<path d="M4 8H28" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-</svg>
+<?xml version="1.0" encoding="utf-8"?><!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
```

Comment:
We should have this [icon](https://www.figma.com/design/cm8L9aMFztSHx91F8MWXQr/Editable-CFA-Site-2023?node-id=62-343&t=S24MrDfMPCeFLCFz-0) in various sizes in the CfA site Figma file... what's up with svgrepo.com?

Reply:
Got it 👍 , I've used the standard icon